tools/bridge/bridge/event_fetcher.py

```diff
@@ -40,9 +41,14 @@ def __init__(
         if event_fetch_limit <= 0:
             raise ValueError("Can not fetch events with zero or negative limit!")
 
-        if max_reorg_depth <= 0:
+        if max_reorg_depth < 0:
             raise ValueError("Invalid maximum reorg depth with zero or negative value!")
```

Comment:
It's not zero, please fix the error message or the test. I don't even know why this has to be changed in this PR.

Reply:
It is possible to set it to zero by the configuration option. I don't see why not. But since you like to have it, I need to introduce another validation function. Afterwards I can "expand" this check again.

Reply:
The error message doesn't match the "less than zero" test. If zero is allowed here, then change the error message.

Reply:
Ah well. Good point!!!

Reply:
Message applied. :heavy_check_mark: